server/views/pages/search/probationSearch/probationSearch.njk

```diff
@@ -24,27 +28,69 @@
     </div>
 {% endmacro %}
 
-{% macro displayResults(countText, results, setPeopleInPrisonTab) %}
-    {% if results.length > 0 %}
+{% macro displayResults(countText, caption, tabId, results, setPeopleInPrisonTab) %}
+    {% set caption = caption %}
+    <h2 class="govuk-heading-l" id = "{{ tabId }}">{{ countText }}</h2>
+    {% if results.length == 0 %}
         {% if setPeopleInPrisonTab %}
-            {% set caption = "In Prison Search Results" %}
-            <h2 class="govuk-heading-l" id = "tab-heading-prison">{{ countText }}</h2>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p>Someone's licence will only appear on this service if they:</p>
+                    <ul class = "govuk-list govuk-list--bullet">
+                        <li>are assigned to you or one of your teams on NDelius</li>
+                        <li>are serving a determinate sentence</li>
+                        <li>have a release date</li>
+                        <li>have not been approved for Home Detention Curfew</li>
+                        <li>have not been recalled or breached a post sentence supervision order</li>
+                    </ul>
+                    <p>If you think someone’s licence is missing, contact the helpdesk by calling 0800 917 5148, or #6598 from inside a prison.</p>
+                    <p class="govuk-body">
+                        {{ govukButton({
+                            text: "Return to caselist",
+                            classes: "govuk-button--secondary govuk-!-margin-bottom-0 govuk-body",
+                            href: backLink,
+                            attributes: { 'data-qa': 'return-to-caselist' }
+                        }) }}
+                    </p>
+                </div>
+            </div>
         {% else %}
-            {% set caption = "On Probation Search Results" %}
-            <h2 class="govuk-heading-l" id = "tab-heading-probation">{{ countText }}</h2>
+            <div class="govuk-grid-row">
```

Comment:
There's quite a lot of duplication in content here.. bit of spot the difference, but I can't see whats the difference between the if/else block?

Reply:
De-duped.